docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -1974,6 +1975,58 @@ Office.context.mailbox.item.getInitializationContextAsync(
 ---
 ---
 
+#### getItemIdAsync([options], callback)
+
+Asynchronously gets the ID of a saved item. Compose mode only.
+
+When invoked, this method returns the item ID via the callback method.
+
+> [!NOTE]
+> If your add-in calls `getItemIdAsync` on an item in compose mode in order to get an `itemId` to use with EWS or the REST API, be aware that when Outlook is in cached mode, it may take some time before the item is actually synced to the server. Until the item is synced, using the `itemId` will return an error.
```

Comment:
```suggestion
> If your add-in calls `getItemIdAsync` on an item in compose mode (e.g., to get an `itemId` to use with EWS or the REST API), be aware that when Outlook is in cached mode, it may take some time before the item is actually synced to the server. Until the item is synced, the `itemId` is not valid and using it will return an error.
```